style.css

```diff
@@ -56,9 +84,18 @@
     height: 0;
   }
 
+  .section-1-1 {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    background-color: #1c1a19;
+    margin-top: -10px;
+    padding-right: 16px;
+    padding-bottom: 80px;
+  }
```

Comment:
Currently, there is an element in your mobile version that is stretching the rest of the content horizontally. Please check the styling fo your mobile version and make sure that it still looks like the figma template🙏

Your design:
<img width="236" alt="image" src="https://user-images.githubusercontent.com/37116322/236555721-1c8bb7c7-fcc6-4649-bd7b-59a2315a1713.png">

Figma design:
<img width="202" alt="image" src="https://user-images.githubusercontent.com/37116322/236555332-aa5b2d25-c8f8-485b-b56a-e4acab48aa91.png">